server.py

```diff
@@ -245,8 +245,12 @@ def update_table():
 
 
 @app.route('/')
-def hello_world():
-    return make_response(jsonify('Hello World'), 200)
+def landing_page():
+    readme = open("README.md", "r")
+    md = markdown.markdown(
+        readme.read(), extensions=["fenced_code"]
+    )
+    return md
```

Comment:
![image](https://user-images.githubusercontent.com/40249949/82858519-f3d9b380-9ec8-11ea-8c27-d1883162fa53.png)
Doesn't seem to capture certain newlines accurately. Not a dealbreaker, but if there's an easy way to fix this we should do it

Reply:
yeah I tried a different extension of nl2br and it fixes the startup formatting issues but then screws up the rest of the file.